addons/delivery/models/delivery_carrier.py

```diff
@@ -115,6 +115,17 @@ def install_more_provider(self):
                 </p>'''),
         }
 
+    def _is_available_for_order(self, order):
```

Comment:
I wonder if we should make this method accept both individual records and recordsets? (same as `available_carriers` below). Would be a bit more in line with `available_carriers` and shorter if we end up reusing it elsewhere).

Reply:
Unless there is an effective gain in performance/clarity/... (e.g. if some operations only have to be done once for all carriers), handling records one at a time is better here IMHO.